utils/imports.js

```diff
@@ -0,0 +1,19 @@
+function ensureImport(j, root, name, source, local) {
```

Comment:
I actually just pulled out a helper method for this here that we can both use: https://github.com/seanjohnson08/ember-getset-util-codemod/commit/b24ebbb5078649cc5db023e86af2e9d0af9b7e2a

Mine accepts an array of names, so you'll have to make that minor change.